src/app/@ansyn/imagery/projection-service/projection.service.ts

```diff
@@ -1,15 +1,25 @@
 import { Observable } from 'rxjs/Observable';
-import { GeometryObject, Point, Position } from 'geojson';
+import { FeatureCollection, GeometryObject, Point, Position } from 'geojson';
 import { IMap } from '@ansyn/imagery/index';
 
 export abstract class ProjectionService {
```

Comment:
We should probably revisit this API in the future. It's not designed well at this moment.

Reply:
👍